style.css

```diff
@@ -0,0 +1,212 @@
+    display: flex; 
+    align-self: flex-end; 
+    right: 0; 
+    width: 94px; 
+    height: 94px; 
+    background: #f2f2f2; 
+    cursor: pointer;
+}
+.header__menu-icon {
+ 	align-self: center; 
+    margin: auto; 
+}
+.menu-line {
+	justify-content:center; 
+    display: block; 
+    width: 35px; 
+    height: 3px; 
+    background: #000; 
+}
+.menu-line:first-of-type {
+	margin-bottom: 5px;
+}
+.wrap {
+    background: linear-gradient(to right, black 491px, white 19%);
+    height: 82vh;
+ }
+.wrap__content {
+	width: 1500px;
+	height: 400px;
+    padding-top: 52px;
+    padding-left: 197px;
+    margin-bottom: 250px;
+}
+.wrap__main-img {
+	position: absolute;
+	width: 900px;
+	height: auto;
+}
+
+.wrap__img {
+	width: 65%;
+	height: auto;
+	padding-top: 0;
+	margin-bottom: 0;
+}
+.wrap__heading {
+	font-family: 'Montserrat-Bold', sans-serif;
+	font-weight: 700;
+	font-size: 86px;
+	line-height: 86px;
+	color: #000000;
+	margin-bottom: 13px;
+	padding-top: 108px;
+	padding-bottom: 8px;
+	padding-left: 643px;
+	width: 500px;
+	margin-top: 0;
+}
+.wrap__article {
+	margin: 0;
+	display: block;
+	font-family: 'Montserrat-Regular', sans-serif;
+	font-size: 16px;
+	line-height: 23px;
+	width: 500px;
+	font-weight: 600;
+	color: #000000;
+	opacity: 0.5;
+	margin-right: 0;
+	margin-bottom: 39px;
+	padding-left: 644px;
+	padding-top: 0;
+}
+.wrap__button-next {
+	font-family: 'Montserrat-Medium', sans-serif;
+	font-size: 14px;
+	text-transform: uppercase;
+	line-height: 23px;
+	font-weight: 500;
+	padding: 12px 32px 29px 33px;
+	width: 218px;
+	height: 47px;
+	border: 1.5px solid #000000;
+	background-color: white;
+	cursor: pointer;
+	margin-bottom: 41px;
+	margin-left: 639px;
+}
+.wrap__button-more {
+	margin: 0;
+	font-family: 'Montserrat-Medium';
+	font-size: 14px;
+	height: 44px;
+	text-transform: uppercase;
+	font-weight: 500;
+	cursor: pointer;
+	text-decoration: none;
+    color: black;
+    padding-left: 53px;
+}
+.button-more__vector {
+	padding-left: 5px;
+}
+
+.footer__social-networks {
+	display: flex;
+	flex-wrap: wrap;
+	margin-left: 10px;
+	height: 43px;
+}
+.footer__social-button {
+	height: 43px;
+	width: 43px;
+}
+.footer__scroll {
+    position: absolute;
+    top: 30px;
+    width: 30px;
+    height: 48px;
+    left: 850px;
+}
+.footer__scroll-link {
+    position: relative;
+    left: auto; 
+}
+.footer__scroll-vector {
+	position: relative;
+	width: 224px;
+	height: 326px;
+	left: 75px;
+	top: 614px;
+}
+@media (max-width: 992px ) {
+
+	.header__logo-img, 
+	.wrap__img,
+	.header__lang-ru,
+	.lang__vector {
+		display: none;
+	}
+	.wrap {
+		background: #fff;
+	}
+
+	.wrap__content {
+		background-image: url(./img/mainpic-mob.png);
+		background-repeat: no-repeat;
+		max-width: 100%;
+		height: 100%;
+    	background-size: cover;
+		display: flex;
+		flex-flow: column;
+		justify-content: flex-end;
+	}
+
+	.header__logo-picture {
+		background-image: url(./img/LOCO-mob.png);
+		width: 98px;
+		height: 37px;
+	}
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=5633355)